plugins/parcel-transformer-markdown-html/package.json

```diff
@@ -9,16 +9,17 @@
 	},
 	"main": "lib/index.js",
 	"engines": {
-		"node": ">= ^20.18.1",
-		"parcel": "^2.12.0"
+		"node": "20.18.1",
+		"npm": "10.8.2"
```

Comment:
For my own edification, what's the motivation for replacing `parcel` with `npm` here?